arviz/plots/plot_utils.py

```diff
@@ -247,3 +247,29 @@ def xarray_to_nparray(data, *, var_names=None, combined=True):
         unpacked_var_names.append(make_label(var_name, selection))
 
     return unpacked_var_names, np.array(unpacked_data)
+
+
+def get_coords(data, coords):
+    """Subselects xarray dataset object to provided coords. Raises exception if fails
+
+    Raises
+    ------
+    ValueError
+        If coords name are not available in data
+
+    KeyError
+        If coords dims are not available in data
+
+    Returns
+    -------
+    data: xarray
+        xarray.Dataset object
+    """
+    try:
+        return data.sel(**coords)
+
+    except ValueError as err:
+        raise ValueError("Verify coords keys. {}".format(err))
```

Comment:
can you verify that the stack trace is still informative? I think the pattern is

```
except ValueError as err:
    raise ValueError("Verify coords keys") from err
```

I might also print out more information - like "make sure {list(coords.keys()} match with the data data_vars ({list(data.data_vars)})"

Reply:
If you do from err the text from the original exception is lost. I was keeping it in because it was informative as to which coordinates were missing. 

But I agree with your second statement which means the above point will be irrelevant! I'll make the changes and post another example.